test/helpers/files.test.ts

```diff
@@ -282,47 +282,43 @@ describe('File Helpers', () => {
   })
 
   describe("cleanCoverageFilePaths()", () => {
+    const ignoreGlobs = fileHelpers.getBlocklist()
+    const paths = fileHelpers.getCoverageFiles(
+      '.',
+      fileHelpers.coverageFilePatterns(),
+    )
```

Comment:
I see what you're doing here. Instead of directly invoking the shared logic here, however, modify `paths` to return the invocation such that the individual tests are still the ones calling `fileHelpers.getCoverageFiles`:

```suggestion
    const paths = () => fileHelpers.getCoverageFiles(
      '.',
      fileHelpers.coverageFilePatterns(),
    )
```

With the above suggestion, `paths` changes ever so slightly. It may make sense to update its name accordingly. `getPaths` makes sense here.

As an aside, it's clear here that `fileHelpers.getCoverageFiles` is not very ergonomic for the (very common) use case of _getting coverage files for the current directory_. Perhaps a future PR can update it to use better defaults.